src/data/equiv/local_equiv.lean

```diff
@@ -501,6 +501,30 @@ end prod
 
 end local_equiv
 
+namespace set
+
+-- All arguments are explicit to avoid missing information in the pretty printer output
+/-- A bijection between two sets `s : set α` and `t : set β` provides a local equivalence
+between `α` and `β`. -/
+noncomputable def bij_on.to_local_equiv [nonempty α] (f : α → β) (s : set α) (t : set β)
```

Comment:
You give the definition, but not the interface. Tagging with `@[simps]` might be a good idea, for instance. Same thing for `inj_on.to_local_equiv`, you might want to register some simp lemmas.

Reply:
Added `simps` and `simp` attrs. It seems that `simps` generates `(hf.to_local_equiv f s).to_fun x = f x`, not `(hf.to_local_equiv f s).to_fun = f`. Is there any reason for this?